src/noises.jl

```diff
@@ -187,8 +189,43 @@ function generate_scenarios(laws::Vector{NoiseLaw}, n::Int64)
     return scenarios
 end
 
+"""
+Simulate n scenarios and return a 3D array
+
+Parameters:
+- laws (Vector{NoiseLaw})
+    Distribution laws corresponding to each timestep
+
+- n (Int64)
+    number of scenarios to simulate
 
+Return:
+- scenarios Array{Float64, 3}
+    scenarios[t,k,:] is the noise at time t for scenario k
 """
+function simulate_scenarios(laws, n::Int64)
+    T = length(laws)
+    dimAlea = size(laws[1].support)[1]
+    dims =(T,n,dimAlea)
+    if typeof(laws) == Distributions.Normal
```

Comment:
sure except that unless I'm mistaken simulate_scenarios(laws, dims) is useless and will be suppressed soon hence the warn("deprecated").